cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -162,6 +200,13 @@ static arrow::Status ScanParquetObject(cls_method_context_t hctx,
   return arrow::Status::OK();
 }
 
+/// \brief The scanning operation to register on Ceph nodes. The request is
+/// deserialized, the object is scanned, and the resulting table is serialized
+/// and sent to the user.
+/// \param[in] hctx Method context.
```

Comment:
when the same parameter like "hctx" is used at multiple places, it is better to keep the descriptions exactly same to prevent confusion

Reply:
Done.